capa/ida/plugin/form.py

```diff
@@ -24,7 +24,8 @@
 from capa.ida.plugin.view import CapaExplorerQtreeView
 from capa.ida.plugin.hooks import CapaExplorerIdaHooks
 from capa.ida.plugin.model import CapaExplorerDataModel
-from capa.ida.plugin.proxy import CapaExplorerSortFilterProxyModel
```

Comment:
renamed existing ProxyModel since we now have two

Reply:
Perfect!